conf/modules/jevois_mavlink.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="jevois_mavlink" dir="sensors">
+  <doc>
+    <description>Send sensor data to jevois and read commands from jevois</description>
+  </doc>
+  <header>
+    <file name="jevois_mavlink.h"/>
+  </header>
+  <init     fun="jevois_mavlink_init()"/>
+  <periodic fun="jevois_mavlink_periodic()" freq="60"/>
+  <periodic fun="jevois_mavlink_filter_periodic()" freq="512"/>
```

Comment:
If it is suppose to run at the main freq, it is probably not necessary to specify it here. Also, this may not work for different main freq or other firmwares.